### ISSUE TITLE
Trackers not linked to harvest for one day

### ISSUE THREAD

**Issue by kaitlyngaynor** [kaitlyngaynor/hopland-hunting#46]
On August 24, 2019, three bucks were taken, but we only have one of them linked to a logger. There is a note "Trackers collected but no numbers linked to 2 bucks shot in James 3 at end of day."

@amcinturff we COULD go through the 15 tracks and see who had clusters in James 3 at "end of day" and potentially figure it out. Tabling it in case we decide it's worth it?

For now, just excluding this day's tracks.

**Comment by kaitlyngaynor** [comment]
In metadata, harvest status is now "unknown" so straightforward to remove

**Comment by kaitlyngaynor** [comment]
E-mailed Alison to ask for locations of harvest for 2019:

> I have one more question about the 2019/2020 data. Did you have hunters report their harvest locations on a map with pins as had been done in previous years? And does it exist as a shape file (or at least as a photograph)? If so, I’d be interested in seeing the points for August 24, 2019, specifically (feel free to send them all over and I can filter from there). There is a note that two deer were harvested in James III late in the day but the harvest isn’t linked to particular loggers, and I’m wondering if I might be able to do some detective work to match the loggers if we have a sense of where the deer were harvested and can see who spent a lot of time near those points late in the day.

**Comment by kaitlyngaynor** [comment]
_From Alex_:

I have completed my investigation.
Many suspects can be ruled out, as they never went to the area, or were not there at "the end of the day." Frustratingly, none of the loggers were there at what I would consider the "end of the day," with the latest being around 3:30-4pm. So, we proceed with that qualification.

There were two pairs of tracks that remain suspects: 2&11, and 6&13. Both appear to be traveling together and were in the area as close to "the end of the day" as anyone. Both have some activity in the area that is suspicious.

My best guess is that logger 6&13 are the murderers.

• Tracks 2 and 11 are in the area around the same time (3:30), but there isn't obvious clustering, and only 11 appears to have exited a vehicle. We know we should be seeing two different clusters in a pair of hunters, but I don't see that here.
• By contrast, there is a very clear behavioral signature of logger 13 near the kill site: get out, walk, stop, make a tight loop back to the car. We see that at a lot of kills
• The other kill site is right on the road, however, so it's not as obvious. It's hard to distinguish a cluster of points from the frequent back and forth traffic on the road. What's more, there's an obvious viewpoint right near this kill site that every logger seemed to stop at for a bit, so that makes it even trickier. Is the cluster I'm seeing a kill or taking in the view? Is there a minimal cluster since it happened on the road and might have been processed extremely fast? ]
• Still, my best guess is that logger 13 (blue track below) murdered near 39.0318802, -123.079906 (brown star below). If that's true, then their companion, logger 6 (yellow track below) most likely murdered at 39.0283004, -123.0772436 (red star below). I think logger 2&11 were in the area at the same time, but do not appear to be the murderers
• This evidence isn't conclusive, but it is the best fitting based on what we know. Alternative explanations could be: the kill site locations are not accurate, the "end of the day" time is not accurate, we are missing hunter or logger data, Alison is misremembering that the murderers were traveling together. Any of these assumptions being wrong would change things.
• All that said, I think this is the best explanation available, based on what we know, and if nothing else, logger 13 (blue track below) murdering near 39.0318802, -123.079906 (brown star below) seems very likely
